### ISSUE TITLE
OnServiceError for Nav and PCM is called even if no service from that type is started

### ISSUE THREAD

**Issue by RHenigan** [smartdevicelink/sdl_java_suite#1574]
### Bug Report
I see these error messages in the log after transport disconnects even if NAV & PCM series are never started:

```
E/BaseEncryptionLifecycleManager: RC_5.0.0: onServiceError, session Type: NAV, reason: Transport disconnected
E/BaseEncryptionLifecycleManager: RC_5.0.0: onServiceError, session Type: PCM, reason: Transport disconnected
```

The reason for the issue is that we call onTransportError() for the aforementioned services without knowing if the service is started or not. This should be tracked so we only call onTransportError when a service is started on these transports.

**Comment by RHenigan** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [(3) onServiceError() for NAV & PCM is called when transport disconnects, even if no service from that type is started](https://trello.com/c/HZK6NbAl/1506-3-onserviceerror-for-nav-pcm-is-called-when-transport-disconnects-even-if-no-service-from-that-type-is-started)

**Comment by JulianKast** [comment]
Fixed with #1575
